pkg/hintrunner/zero/zerohint_usort.go

```diff
@@ -26,3 +30,63 @@ func newUsortEnterScopeHint() hinter.Hinter {
 func createUsortEnterScopeHinter() (hinter.Hinter, error) {
 	return newUsortEnterScopeHint(), nil
 }
+
+func newUsortVerifyMultiplicityBodyHint(nextItemIndex hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "UsortVerifyMultiplicityBody",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> current_pos = positions.pop()
+			//> ids.next_item_index = current_pos - last_pos
+			//> last_pos = current_pos + 1
+
+			positionsInterface, err := ctx.ScopeManager.GetVariableValue("positions")
+			if err != nil {
+				return fmt.Errorf("getting positions from scope failed: %w", err)
+			}
+
+			positions, ok := positionsInterface.([]uint64)
+			if !ok {
+				return fmt.Errorf("casting positions into an array of uint64 failed: %w", err)
+			}
+
+			current_pos := utils.Pop(&positions)
+			last_pos := current_pos + 1
```

Comment:
You're right
I was confused because it didn't make sense to me that we were using a variable that was not declared but now that I understand more the code I noticed that `last_pos` is present in the scope and we just need to get the value from there

Thanks for pointing this out @MaksymMalicki